docs/language-agnostic-definition.md

```diff
@@ -0,0 +1,99 @@
+An Autonomous Economic Agent is, in technical terms, defined by the following characteristics:
+
+- It MUST be capable of receiving and sending `Envelopes` which satisfy the following protobuf schema:
+
+``` proto
+syntax = "proto3";
+
+package fetch.aea;
+
+message Envelope{
+    string to = 1;
+    string sender = 2;
+    string protocol_id = 3;
+    bytes message = 4;
+    string uri = 5;
+}
+```
+
+<div class="admonition note">
```

Comment:
@marcofavorito shall we add the regex on to, from and protocol_id and uri?

Reply:
Not sure. We have to document somewhere, for sure; but maybe this is not the right place.
It depends on what is the purpose of this document: is it a user-documentation (hence more accessible and readable), or a specification, e.g. RFCs? In the former case, I'd omit it.

Anyway, not much work required for adding it.

Reply:
It is replacing an RFC style doc until we have one.

Reply:
What are these regexes?

Reply:
most of them TBD; the only one we have is the one for the `protocol_id` which can be found in definitions under configurations/schema